### DISCUSSION TITLE
Table Width

### DISCUSSION THREAD

**Discussion by vibe-code-repo** [icereed/paperless-gpt#8] [Ideas]
Hey, an idea / suggestion for improvement. Is it possible to set the table to 100% so that the space is utilized better? It would also be great if one could define the column width.

**Comment by icereed** [comment]
Hey @mikekaldig, thanks for your input. Yeah, the table wasn't looking great. However, how do you like this cards based design?
It's in this PR: https://github.com/icereed/paperless-gpt/pull/11

**Comment by vibe-code-repo** [comment]
I like this better, but the hover effect for the text display is a bit too small in my opinion. This requires a lot of scrolling, especially with longer texts. Would it be possible to add an "image icon" that shows the document as a preview, similar to Paperless?
